openrpc/src/schemas/Pagination.json

```
@@ -1,16 +1,28 @@
 {
-    "Pagination": {
+    "EventsPagination": {
         "type": "object",
         "properties": {
             "cursor": {
-                "type": "string",
-                "description": "A string ID that points to a specific location in a collection of responses and is pulled from the `paging_token` value of a record. When a cursor is provided Soroban-RPC will _not_ include the element whose id matches the cursor in the response. Only elements which appear _after_ the cursor are included."
+                "$ref": "#/components/schemas/Cursor"
             },
             "limit": {
                 "type": "number",
                 "description": "The maximum number of records returned. The limit for [getEvents](https://developers.stellar.org/network/soroban-rpc/getEvents) can range from 1 to 10000 - an upper limit that is hardcoded in Soroban-RPC for performance reasons. If this argument isn't designated, it defaults to 100."
             }
         },
         "required": []
+    },
+    "TransactionsPagination": {
+        "type": "object",
+        "properties": {
+            "cursor": {
+                "$ref": "#/components/schemas/Cursor"
+            },
+            "limit": {
+                "type": "number",
+                "description": "The maximum number of records returned. The limit for [getTransactions](https://developers.stellar.org/network/soroban-rpc/getTransactions) can range from 1 to 200 - an upper limit that is hardcoded in RPC for performance reasons. If this argument isn't designated, it defaults to 50."
```

Too detailed for public API docs

```suggestion
                "description": "The maximum number of records returned. For [getTransactions](https://developers.stellar.org/network/soroban-rpc/getTransactions), this ranges from 1 to 200 and defaults to 50."
```

(Isn't there a `default` for OpenAPI specs?)

---

Ohh I did not know about any default OpenAPI specs. Let me check